app/controllers/index.rb

```diff
@@ -0,0 +1,147 @@
+enable :sessions
+
+get '/' do
+  redirect '/user' if session[:user_id]
+    @da = Station.find(13)
```

Comment:
This mapping seems very important. Maybe it should live in the `Station` modal? Also it is not clear what it means.